app/Http/Controllers/AnswerController.php

```diff
@@ -55,6 +57,9 @@ public function store(Request $request)
 			$notification->notifyFormCreatorWhatsapp($form,  $respondent);
 			$notification->notifyFormCreatorWebhook($form,  $respondent);
 			$notification->notifyFormRespondentEmail($form,  $respondent);
+
+			$updateTimeMetric = new MetricsService();
+			$updateTimeMetric->updateFormTime($answer);
```

Comment:
```suggestion
			(new MetricsService())->updateFormTime($answer);
```

Como você está manipulando uma única coisa, acredito que pode fazer dessa forma. Só estilo, nada errado como tu fez.